pg.js

```diff
@@ -948,7 +948,7 @@ function WHERE(builder, scalar, group, params) {
 				break;
 			case 'date':
 				opuse && condition.length && condition.push(op);
-				condition.push(cmd.name + '::date' + cmd.compare + cmd.value instanceof Date ? (cmd.value.format('yyyy-MM-dd') + '::date') : 'null');
+				condition.push(cmd.name + '::date' + cmd.compare + (cmd.value instanceof Date ? (ESCAPE(cmd.value.format('yyyy-MM-dd')) + '::date') : 'null'));
```

Comment:
Prečo `ECAPE()`?

Reply:
lebo on tam pridá znak ' (bez neho to bralo ako column name)
čiže to urobí 

`dtcreated::date = '2021-06-17'::date`

Reply:
A nie je lepšie ten znak pridať priamo do formátu?

Reply:
NOW.format('\'yyyy-MM-dd\'')

hodí error

Reply:
`NOW.format('\'yyyy-MM-dd\'')`
\'

Reply:
A toto `NOW.format('\'yyyy-MM-dd\'')`?

Reply:
áno presne to hodí error, github dal preč znak `\` (to bolo prvé čo mi napadlo, rovno vo formate úvudzovky)

Reply:
Pozriem to.